model/project.go

```diff
@@ -1906,10 +1906,22 @@ func (p *Project) extractDisplayTasks(pairs TaskVariantPairs) TaskVariantPairs {
 			displayTV := TVPair{Variant: bv.Name, TaskName: dt.Name}
 			execTVPairs := make([]TVPair, 0, len(dt.ExecTasks))
 			for _, et := range dt.ExecTasks {
-				exexTV := TVPair{Variant: bv.Name, TaskName: et}
-				execTaskToDisplayTask[exexTV] = displayTV
-				execTVPairs = append(execTVPairs, exexTV)
+				if len(et) > 0 && et[0] == '.' {
+					// If this execution task starts with a period, it is actually task tag.
+					tagToFind := et[1:]
```

Comment:
I'm maybe a little confused by what this PR fixes, but aren't execution tasks already resolved from tag selectors during project translation? While I do think display tasks should be scheduled alongside execution tasks, is it necessary to expand execution task tags here? It seems like [the parser project already should have resolved execution tasks tag selectors during project translation](https://github.com/evergreen-ci/evergreen/blob/d9876e5ba1a93b46ed46525987a4dcad5833fe2a/model/project_parser.go#L1280), so I would have expected the `Project`'s execution task lists to already have the resolved full list of real execution tasks at this point, not tags.

Reply:
Hm I think you're right. I re-tested right now before and after and actually the before finalized patch gives the same result- since it does translate it. I think I was comparing one that I scheduled on the configure page vs one that is actually finalized. I think we should close this ticket in favor of the UI ticket I made then, what do you think? 

It makes sense why via the test it doesn't work since the project is suppose to already be translated and not have tags in that section.

Reply:
Is the issue specific to the UI? As in if you finalize with all tasks/variants from the CLI (or only schedule the execution task), then does Evergreen create the display task automatically?

Reply:
Yes it's only with the UI, finalizing doesn't make it happen